riscv/insns/c_sspopchk_x5.h

```diff
@@ -0,0 +1,32 @@
+reg_t prv = STATE.prv;
+reg_t xSSE = 0;
+
+if (prv == PRV_M)
+  xSSE = 0;
+else if (prv == PRV_S || prv == PRV_HS)
+  xSSE = get_field(STATE.menvcfg->read(), MENVCFG_SSE);
+else if (p->extension_enabled('S'))
+  xSSE = get_field(STATE.senvcfg->read(), SENVCFG_SSE);
+else
+  xSSE = false;
+
```

Comment:
Could we put this into decode_macros as this is used by multiple instructions?
get_xSSE()?

Reply:
We can create a macro like this
```
#define SET_XSSE(xSSE) \
  reg_t prv = STATE.prv; \
  if (prv == PRV_M) \
    xSSE = false; \
  else if (prv == PRV_S || prv == PRV_HS) \
    xSSE = (get_field(STATE.menvcfg->read(), MENVCFG_SSE) == 1); \
  else if (p->extension_enabled('S')) \
    xSSE = (get_field(STATE.senvcfg->read(), SENVCFG_SSE) == 1); \
  if (prv == PRV_S && STATE.v) \
    xSSE = xSSE & (get_field(STATE.henvcfg->read(), HENVCFG_SSE) == 1);
```